frontend/components/homepage/Footer.js

```diff
@@ -9,7 +9,7 @@ function Copyright() {
     <Typography variant="body2" color="text.secondary" align="center">
```

Comment:
Footer.js can be removed, because we have lots of negative feedbacks about it but not crucial.